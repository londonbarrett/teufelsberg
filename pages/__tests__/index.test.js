/* eslint-env jest */

import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Index from '../index.js';

describe('Index Unit Enzyme', () => {
  it('Home shows "Index Page"', () => {
    const index = shallow(<Index />);

    expect(index.find('h1').text()).toEqual('Index Page');
  });
});

describe('Index Snapshot Testing', () => {
  it('Renders Index Page correctly', () => {
    const component = renderer.create(<Index />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
