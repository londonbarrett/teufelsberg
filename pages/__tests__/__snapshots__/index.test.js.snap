// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Snapshot Testing Renders Index Page Correctly 1`] = `
<main
  className="jsx-3844205166"
>
  <article
    className="jsx-3844205166"
  >
    <header
      className="jsx-56047622 header"
    >
      <h1
        className="jsx-56047622 title"
      >
        
      </h1>
    </header>
    <section
      className="jsx-1845042281 container"
    />
    <section
      className="jsx-1346030082 container"
    >
      <div
        className="jsx-1346030082 header"
      >
        <span
          className="jsx-1346030082 title"
        >
          
        </span>
        <a
          className="jsx-348398248 link"
          href="/"
          onClick={[Function]}
        >
          X
        </a>
      </div>
      <div
        className="jsx-1346030082 content"
      >
        <div
          className="jsx-4014059616"
        >
          <img
            alt=""
            className="jsx-4014059616 image"
            src="https://image_01_27.jpg"
          />
        </div>
      </div>
    </section>
  </article>
</main>
`;
